tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKBuildPropertiesResourceListener.java

```diff
@@ -44,40 +46,41 @@
 public class SDKBuildPropertiesResourceListener implements IResourceChangeListener, IResourceDeltaVisitor
 {
 
-    private final static String MARKER_ID_SDK_PROPERTIES_INVALID = "sdk-properties-invalid";
+    private static final String ID_SDK_PROPERTIES_INVALID = "sdk-properties-invalid";
+
 
     private final static Pattern PATTERN_BUILD_PROPERTIES  = Pattern.compile("build.[\\w|\\W.]*properties");
 
-    protected void processPropertiesFileChanged() throws CoreException
+    protected void processPropertiesFileChanged( final IFile deltaFile ) throws CoreException
     {
-        final SDK sdk = SDKUtil.getWorkspaceSDK();
 
-        if( sdk != null )
+        IProject deltaProject = deltaFile.getProject();
+
+        final SDK sdk = SDKUtil.createSDKFromLocation( deltaProject.getLocation() );
+
+        if ( sdk != null )
         {
-            final IProject workspaceSDKProject = SDKUtil.getWorkspaceSDKProject();
+            final IProject sdkProject = SDKUtil.getWorkspaceSDKProject();
```

Comment:
you changed the variable name from "workspaceSDKProject" back to "sdkProject"  this undos my change that I made previously.  Perhaps you copied an older version of the file?  I'm wondering why you changed this variable name back?